gensim/test/test_atmodel.py

```diff
@@ -37,7 +37,10 @@
 # Test that models are compatiple across versions, as done in LdaModel.
 
 module_path = os.path.dirname(__file__)  # needed because sample data files are located in the same folder
-datapath = lambda fname: os.path.join(module_path, 'test_data', fname)
+
+
+def datapath(fname):
```

Comment:
please create `utils` file and move this function (replace to import everywhere).